Source/MarkdownGenerator/src/main/java/net/steppschuh/markdowngenerator/list/OrderedList.java

```diff
@@ -0,0 +1,29 @@
+package net.steppschuh.markdowngenerator.list;
+
+import java.util.List;
+
+public class OrderedList extends UnorderedList<OrderedListItem> {
```

Comment:
Please stay consistent with `UnorderedList` and use a generic item type

```suggestion
public class OrderedList<T extends Object> extends UnorderedList<T> {
```

Reply:
@Revxrsal 
Do you plan to respond to this one?